cssdb.json

```
@@ -526,6 +526,15 @@
       }
     ]
   },
+  {
+    "id": "prefers-reduced-motion",
+    "title": "`prefers-reduced-motion` Media Query",
+    "description": "Media query based on a user preference for preferring reduced motion (animation, etc)",
```

Could we describe this as the following?

```suggestion
    "description": "A media query to detect a user’s desire for less animation and general motion on the page",
```

---

Thank you for helping with texts!